### PR TITLE
Add padig:mediaMaster to Funnel Cake Solr indexing

### PR DESCRIPTION
**Pull request by ebtoner**
See https://github.com/tulibraries/aggregator_mdx/pull/204. New field mapped in transform, which we would like to display in Funnel Cake for QA. Once this is merged, I will add to the Blacklight controller.

### CODE REVIEW

**Comment by ebtoner** [comment]
@dkinzer @nomadicoder Is there an issue with Travis CI here? I know we've been moving away from Travis to Circle CI over time. These tests don't seem to be completing...

**Review by dkinzer — APPROVED** [review]

**Comment by dkinzer** [comment]
@ebtoner Sorry, I think travis is busted.  I need to move these to CircleCI. Will do that on Monday.

File: lib/funnel_cake_index/indexer_config.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@
           "schema" => "http://schema.org",
           "oai_qdc" => "http://worldcat.org/xmlschemas/qdc-1.0/",
           "svcs" => "http://rdfs.org/sioc/services"
```
Comment (dkinzer) [inline lib/funnel_cake_index/indexer_config.rb:33]: @ebtoner this needs a comma here or it will fail.

**Review by dkinzer — CHANGES_REQUESTED** [review]
This is failing locally without the comma.

**Review by dkinzer — APPROVED** [review]

**Pull request merged**